mkdocs/pages/en/devbook/accounts/create-from-private-key.md

```diff
@@ -78,6 +77,9 @@ This tutorial showed how to:
 | [Get the public key](#creating-the-account)                   | <dy:KeyPair.publicKey>                   |
 | [Get the address](#creating-the-account)                      | <dy:network.publicKeyToAddress>          |
 
-## Next steps
+## Next Steps
 
-Once you have created an account, you can use it to [send transactions](../transactions/transfer.md).
```

Comment:
The link to the faucet tutorial is a good addition, but I would keep the other one too. We can have multiple next steps.